gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -870,6 +870,54 @@ public static SnapshotFilter notEquals(SnapshotField field, long value) {
   }
 
   /**
+   * Class for filtering image lists.
+   */
+  class ImageFilter extends ListFilter {
+
+    private static final long serialVersionUID = -3601427417234098397L;
+
+    private ImageFilter(ImageField field, ComparisonOperator operator, Object value) {
+      super(field.selector(), operator, value);
+    }
+
+    /**
+     * Returns an equals filter for the given field and string value. For string fields,
+     * {@code value} is interpreted as a regular expression using RE2 syntax. {@code value} must
+     * match the entire field.
+     *
+     * @see <a href="https://github.com/google/re2/wiki/Syntax">RE2</a>
+     */
+    public static ImageFilter equals(ImageField field, String value) {
+      return new ImageFilter(checkNotNull(field), ComparisonOperator.EQ, checkNotNull(value));
+    }
+
+    /**
+     * Returns a not-equals filter for the given field and string value. For string fields,
+     * {@code value} is interpreted as a regular expression using RE2 syntax. {@code value} must
+     * match the entire field.
+     *
+     * @see <a href="https://github.com/google/re2/wiki/Syntax">RE2</a>
+     */
+    public static ImageFilter notEquals(ImageField field, String value) {
+      return new ImageFilter(checkNotNull(field), ComparisonOperator.NE, checkNotNull(value));
+    }
+
+    /**
+     * Returns an equals filter for the given field and long value.
+     */
+    public static ImageFilter equals(ImageField field, long value) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The method name and parameter number are suspiciously close to equals(Object)](https://www.codacy.com/app/mziccard/gcloud-java/file/2341145921/issues/source?bid=3140968&fileBranchId=3207740#l908)